packages/shortest/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@antiwork/shortest",
-  "version": "0.1.1",
+  "version": "0.2.1",
```

Comment:
Changed the version to 0.2.1 to indicate that this version introduces a new feature. This would be more consistent with semantic versioning best practices. 

![image](https://github.com/user-attachments/assets/81b36c84-f135-4079-b37c-6e6cfd9990c4)